app/controllers/group_assignments_controller.rb

```diff
@@ -43,6 +45,26 @@ def show
       .page(params[:students_page])
   end
 
+  # rubocop:disable MethodLength
+  def filter_repos
+    return unless search_assignments_enabled?
+    matching_groups = @group_assignment.grouping.groups
+    if @query.present?
+      matching_groups = matching_groups.where("slug LIKE ?", "%#{@query}%")
```

Comment:
Can we search by the team name and not the slug? Searching for `Team` or `A` with a team called `Team A` returns no results since the slug is `team-a`.